src/select/index.tsx

```diff
@@ -229,7 +230,9 @@ export const Select = factory(function Select({
 							close();
 						}
 
-						return (
+						return isLoading(getOptions()) ? (
```

Comment:
I think this should only show if the resource is loading and the current total is undefined. Otherwise this would show the loading indicator instead of results when the resource is fetching the next page etc.